<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>453</width>
    <height>254</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Scan Photo</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,0,0,0">
    <item row="1" column="3">
     <widget class="QComboBox" name="comboColor"/>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Color Mode :</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Resolution :</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QLabel" name="labelExt">
      <property name="text">
       <string>  .jpg</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Scan Area :</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QComboBox" name="comboArea"/>
    </item>
    <item row="2" column="3">
     <widget class="QComboBox" name="comboResolution">
      <property name="currentIndex">
       <number>-1</number>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Output :</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLineEdit" name="filenameEdit">
      <property name="placeholderText">
       <string>filename</string>
      </property>
     </widget>
    </item>
    <item row="5" column="3" colspan="2">
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="scanBtn">
         <property name="text">
          <string>Scan</string>
         </property>
         <property name="icon">
          <iconset theme="scanner"/>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="closeBtn">
         <property name="text">
          <string>Close</string>
         </property>
         <property name="shortcut">
          <string>Esc</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0" rowspan="4">
     <widget class="QLabel" name="scannerLabel">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="../files/resources.qrc">:/scanner.png</pixmap>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QComboBox" name="comboDevice"/>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../files/resources.qrc"/>
 </resources>
 <connections/>
</ui>
